lsp-vhdl.el

```diff
@@ -72,26 +72,26 @@ HDL Checker: A wrapper for third party tools such as GHDL, ModelSim, Vivado Simu
 
 (defun lsp-vhdl--create-connection ()
   "Returns lsp-stdio-connection based on the selected server"
-  (setq params nil)
+  (defvar lsp-vhdl--params nil)
```

Comment:
defvar should be on the top level and then you should use setq in the method.

Reply:
What exactly is top level in Lisp? Is it the top of the file or right after `(lsp-register-client` in line 95 in this case? Or can you point me to another example where support for multiple servers have been implemented? I feel like I'm reimplementing the wheel where others might have done something similar.

Reply:
> What exactly is top level in Lisp?

It is outside of the function, like `defcustom`.

> Or can you point me to another example where support for multiple servers have been implemented? 

We usually have lsp-register-client per server(check lsp-rust.el for example).

Reply:
I have moved the defvar outside the function for now. I will look into refactoring the code at some point so it will be more similar to how other people have done it.